ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -190,11 +192,16 @@ uint32_t RSkTextLayoutManager::buildParagraph (SharedColor backGroundColor,
         style.setDecoration(fragment.textAttributes.textDecorationLineType.has_value() ?
                                 convertDecoration(fragment.textAttributes.textDecorationLineType.value()) :
                                 TextDecoration::kNoDecoration);
+        style.addShadow(shadow);
         style.addShadow(TextShadow(convertTextColor(fragment.textAttributes.textShadowColor ?
                                                     fragment.textAttributes.textShadowColor :
                                                     fragment.textAttributes.foregroundColor).getColor(),
                                                     setShadowPoint.Make(fontShadowOffset.width,fontShadowOffset.height),
                                                     fontShadowRadius));
+        if(!backGroundColor) {
```

Comment:
having a question , Will the container shadow to be added first or textshadow to be added first ?

What are these two addition add shadow is for ? Is both for content ?

Reply:
First container shadow then textshadow of shadow and last textshadow to be added. First add shadow for contain shadow second add shadow for textshadow of shadow .and 3rd add shadow is for textshadow.